journey/pepr-build.ts

```diff
@@ -1,23 +1,45 @@
 // SPDX-License-Identifier: Apache-2.0
 // SPDX-FileCopyrightText: 2023-Present The Pepr Authors
 import { loadAllYaml } from "@kubernetes/client-node";
-import { expect, it } from "@jest/globals";
+import { afterAll, beforeAll, expect, it } from "@jest/globals";
 import { loadYaml } from "@kubernetes/client-node";
-import { execSync } from "child_process";
+import { exec, execSync } from "child_process";
 import { promises as fs } from "fs";
 import { resolve } from "path";
 import { V1ObjectMeta, KubernetesObject } from '@kubernetes/client-node';
 import yaml from "js-yaml";
 import { cwd } from "./entrypoint.test";
 
 export function peprBuild() {
+
+  const moduleName = "pepr-test-module"
+
+  beforeAll(() => {
+    console.info("!!!STARTING PEPR-BUILD TESTS!!!")
+    execSync(`jq '.dependencies.pepr = "file:../0.0.0-development"' package.json > temp.json && mv temp.json package.json`, {cwd: 'pepr-test-module'})
+    execSync('npm install', {cwd: 'pepr-test-module'})
+    console.log(execSync(`jq '.dependencies' package.json`, {cwd: 'pepr-test-module'}).toString())
+
+    //Prepare the 'env' key in the test module's package.json
+    const envValues = '{"MY_CUSTOM_VAR": "example-value","ZARF_VAR": "###ZARF_VAR_THING###"}'
+    execSync(`jq \'.pepr.env = ${envValues}\' package.json > temp.json && mv temp.json package.json`, {cwd: 'pepr-test-module'});
```

Comment:
```suggestion
    execSync(`jq \'.pepr.env = ${envValues}\' package.json > temp.json && mv temp.json package.json`, {cwd: 'pepr-test-module'});
```

Will this work on Windows?